code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -40,10 +40,6 @@
 		return
 	if(!AStar(user, target.loc, /turf/proc/Distance, user.mind.changeling.sting_range, simulated_only = 0))
 		return
-	if(target.mind && target.mind.changeling)
-		sting_feedback(user,target)
-		take_chemical_cost(user.mind.changeling)

```

Comment:
This wasn't getting called twice, because this wasn't returning true; it exists to not affect other changelings with stings while still taking the cost.